lib/promscrape/scrapework_test.go

```diff
@@ -850,6 +873,19 @@ func timeseriesToString(ts *prompbmarshal.TimeSeries) string {
 	}
 	s := ts.Samples[0]
 	fmt.Fprintf(&sb, "%g %d", s.Value, s.Timestamp)
+	// Add Exemplars to the end of string
+	for j, exemplar := range ts.Exemplars {
+		for i, label := range exemplar.Labels {
+			fmt.Fprintf(&sb, "%s=%q", label.Name, label.Value)
+			if i+1 < len(ts.Labels) {
+				fmt.Fprintf(&sb, ",")
+			}
+		}
+		fmt.Fprintf(&sb, "%g %d", exemplar.Value, exemplar.Timestamp)
```

Comment:
Should it always append timestamp to the string?
If timestamp was not present it will have a default value of 0. Seems like it will be better to handle this case and do not append `0` value.

Reply:
I copied line 875 which is how it's done for the samples where timestamp is also optional.